src/components/dialog/bl-dialog.css

```diff
@@ -1,36 +1,62 @@
-dialog {
-  padding: 0;
-  border: 0;
+.container {
+  display: flex;
+  flex-direction: column;
   background: var(--bl-color-primary-background);
-  border-radius: var(--bl-border-radius-l);
   max-width: calc(100vw - var(--bl-size-4xl));
   max-height: calc(100vh - var(--bl-size-4xl));
-  overflow: hidden;
+  min-width: 424px;
+  padding: 0;
+  border: 0;
+  border-radius: var(--bl-border-radius-l);
 }
 
-dialog::backdrop {
+.dialog,
+.dialog-polyfill .container {
+  padding: 0;
+  border: 0;
+  border-radius: var(--bl-border-radius-l);
+}
+
+.dialog-polyfill .container {
+  position: fixed;
+  z-index: 999; /* FIX ME -> Use z-index variable */
```

Comment:
```suggestion
  /* FIXME: Use z-index variable */
  z-index: 999;
```

Reply:
Is this a general rule ? The linter expects empty line before comment. So this looks like this:


![image](https://user-images.githubusercontent.com/26359615/202983165-a7975643-3472-4227-ac67-5431592da58a.png)

Reply:
Yeah, I think it's for making it more noticeable. But still, I think it should also be like `FIXME:` instead of `FIX ME ->`